source/Pages/Notification/components/Sign/components/WarningModal/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { Button } from '@ui';
+
+import useStyles from './styles';
+
+const WarningModal = ({ toggleModal }) => {
```

Comment:
I think it'd be worth to reuse some MUI built-in things to do modals
For now it's totally fine to have it, but MUI do some extra magic like react portals, accessibility, keyboard clicks and stuff like that

Reply:
🤮 MUI imho lol 
cc @angarita-dev back me up hahahah

Reply:
Some translations are missing :)